src/main/java/com/creativemd/littletiles/common/items/ItemBlockTiles.java

```diff
@@ -33,11 +31,8 @@
 import com.creativemd.littletiles.common.packet.LittlePlacePacket;
 import com.creativemd.littletiles.common.structure.LittleStructure;
 import com.creativemd.littletiles.common.tileentity.TileEntityLittleTiles;
-import com.creativemd.littletiles.common.utils.LittleTile;
+import com.creativemd.littletiles.common.utils.*;
```

Comment:
Please avoid star imports

Reply:
Sorry, that's from before I reconfigured Idea to not use them :sweat_smile: